starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,76 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <title>Apple Pie Recipe</title>
-</head>
-<body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
-
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
-
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
-flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce
-temperature and let simmer.
-
-Place the bottom crust in your pan. Fill with apples, mounded slightly. Cover with a
-lattice work crust. Gently pour the sugar and butter liquid over the crust. Pour slowly so
-that it does not run off.
-
-Bake 15 minutes in the preheated oven. Reduce the temperature to 350 degrees F (175
-degrees C). Continue baking for 35 to 45 minutes, until apples are soft.
-</body>
+  <head>
+
+    <title>Apple Pie Recipe</title>
+    <link rel="stylesheet" href="./css/style.css" />
+
+  </head>
+
+  <body>
+    <header class="header">
+      <h1>Apple Pie</h1>
+    </header>
+
+    <div class="content">
+      <section class="Apple pie history">
+        <p>
+          This was my grandmother's apple pie recipe. I have never seen another
+          one quite like it. It will always be my favorite and has won me
+          several first place prizes in local competitions. I hope it becomes
+          one of your favorites as well!
+        </p>
+        
+        <img src="images/apple-pie.jpg" alt="apple-pie">
+            
+      </section>
+  
+      <section class="ingredients">
+        <h3>Ingredients</h3>
+        <ol class="ingredients list">
+          <li>- 1 recipe pastry for a 9 inch double crust pie</li>
+          <li>- 1/2 cup unsalted butter</li>
+          <li>- 3 tablespoons all-purpose flour</li>
+          <li>- 1/4 cup water</li>
+          <li>- 3 tablespoons all-purpose flour</li>
+          <li>- 1/4 cup water</li>
+          <li>- 1/2 cup white</li>
+          <li>- sugar 1/2 cup packed brown sugar</li>
+          <li>- 8 Granny Smith apples - peeled, cored and sliced</li>
+        </ul>
+        
+        <img src="images/recipe-info" alt="recipe-info">
```

Comment:
You were really close! 
`      <img src="./images/recipe-info.png" alt="recipe-info">` You needed to add the .png at the end